src/main/java/com/tttbotapi/tictactoebotapi/controller/BotController.java

```diff
@@ -1,13 +1,61 @@
 package com.tttbotapi.tictactoebotapi.controller;
 
 import com.tttbotapi.tictactoebotapi.model.Board;
+import com.tttbotapi.tictactoebotapi.model.Bot;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.HashMap;
+
+/**
+ * This class serves as a REST API controller for Tic Tac Toe game.
+ * <p>
+ * It exposes endpoints to play the game by receiving the symbol of the player
+ * <p>
+ * with the board and responding with the current board with the bot move and game status.
+ *
+ * @RestController - Indicates that this class is a REST controller.
+ * @RequestMapping(path = "/api") - Specifies the base URI path for the controller's endpoints.
+ */
 @RestController
 @RequestMapping(path = "/api")
 public class BotController {
-    @PutMapping(path = "/{symbol}")
-    public Board getInfo(@PathVariable(name = "symbol") String symbol, @RequestBody Board inputBoard) {
-        return ;
+    // Instance variables to store the Bot and Board objects.
+    private Bot bot = new Bot();
+    public Board board = new Board();
```

Comment:
I think we don't really need to instantiate it, because an instance is already created when we are receiving a POST request and makeMove is also returning an instance.